src/Products/Viewer/Util/DocumentDataLoader.cs

```diff
@@ -0,0 +1,363 @@
+﻿using GroupDocs.Viewer.Caching;
+using GroupDocs.Viewer.Exceptions;
+using GroupDocs.Viewer.MVC.Products.Common.Entity.Web;
+using GroupDocs.Viewer.MVC.Products.Common.Resources;
+using GroupDocs.Viewer.MVC.Products.Viewer.Cache;
+using GroupDocs.Viewer.Options;
+using GroupDocs.Viewer.Results;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Web;
+using System.Xml.Linq;
+
+namespace GroupDocs.Viewer.MVC.Products.Viewer.Util
+{
+    public class DocumentDataLoader
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5958475)